lib/config_cat.ex

```diff
@@ -172,6 +186,8 @@ defmodule ConfigCat do
           | {:cache_policy, CachePolicy.t()}
           | {:data_governance, data_governance()}
           | {:http_proxy, String.t()}
+          | {:connect_timeout, Integer.t()}
+          | {:read_timeout, Integer.t()}
```

Comment:
I missed this in my earlier review. These types (and below) should be `integer()` or `non_neg_integer()` probably. There is no such thing as `Integer.t()` in Elixir.

This is the root cause of the current CI failures.

See https://hexdocs.pm/elixir/1.13/typespecs.html for more details.